src/sentry/api/endpoints/user_emails.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+from django.db import IntegrityError, transaction
+from django.utils.translation import ugettext as _
+from rest_framework import serializers, status
+from rest_framework.response import Response
+
+from sentry.api.bases.user import UserEndpoint
+from sentry.models import UserEmail
+
+
+class UserEmailSerializer(serializers.Serializer):
+    email = serializers.EmailField(required=True)
+    primary = serializers.BooleanField(required=False, default=True)
+
+
+class UserEmailsEndpoint(UserEndpoint):
```

Comment:
kinda wondering if this would make more sense as part of the `UserDetailsEndpoint` rather than its own endpoint. I think that'd be fine for something like this, where you're adding emails, but might get tricky for email deletion, which we'll eventually have to implement when we move the account settings to react

@mattrobenolt what do you think?

Reply:
I don't see any downside to having it as its own endpoint, but I'll defer to whatever Matt thinks. It'd be nice to get this approved by the time we get back—I have another PR that's dependent on this.